static/src/stylesheets/module/commercial/_adverts-soulmates.scss

```diff
@@ -1,5 +1,7 @@
 .adverts--soulmates {
     @include mq($until: tablet) {
+        // In the soulmates component, we want to show rows of 3 profiles
```

Comment:
Good code is its own best documentation. As you’re about to add a comment, ask yourself, ‘How can I improve the code so that this comment isn’t needed?